src/config/telemetry-config.json

```diff
@@ -3,6 +3,10 @@
         "enabled":"all",
         "refresh": "12h",
         "includes": [
+            {
+                "name" : "startup",
+                "dailyLimit": 1
+            },
             {
                 "name" : "*"
             }
```

Comment:
wildcard pattern needs to be set last, so specialized patterns can match first

Reply:
I'm putting that one on the backburner as we might need to restrict telemetry to a single event, without having to exclude all the others individually.